packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -478,6 +531,7 @@ class SliderThemeData extends Diagnosticable {
       valueIndicatorTextStyle: defaultTheme.accentTextTheme.body2,
     );
     properties.add(DiagnosticsProperty<Color>('activeTrackColor', activeTrackColor, defaultValue: defaultData.activeTrackColor));
+    properties.add(DiagnosticsProperty<Color>('activeTrackColor', activeTrackColor, defaultValue: defaultData.activeTrackColor));
```

Comment:
Please add a debugFillProperties() test, like https://github.com/flutter/flutter/blob/master/packages/flutter/test/material/outline_button_test.dart#L274

Reply:
Can this be a follow up PR, since there is no existing slider debugFillProperties  test, and this PR is already large?

Reply:
Sure